File: articles/documentation-model-cards-and-datasheets-for-algorithms/go/documentation_quality_score.go
package main

import "fmt"

func main() {
	scores := []float64{0.62, 0.6875, 0.58, 0.50, 0.56, 0.52}
	total := 0.0
	for _, score := range scores {
		total += score
	}
	fmt.Printf("documentation_quality_score=%.4f\n", total/float64(len(scores)))
}

File: articles/documentation-model-cards-and-datasheets-for-algorithms/java/DocumentationQualityScore.java
public class DocumentationQualityScore {
    public static void main(String[] args) {
        double[] scores = {0.62, 0.6875, 0.58, 0.50, 0.56, 0.52};
        double total = 0.0;
        for (double score : scores) total += score;
        System.out.printf("documentation_quality_score=%.4f%n", total / scores.length);
    }
}
